src/sentry/digests/utilities.py

```diff
@@ -23,47 +31,40 @@ def get_digest_metadata(digest):
     return start, end, counts
 
 
-def get_personalized_digests(target_type, project_id, digest, user_ids):
+def get_personalized_digests(
+    target_type: ActionTargetType, project_id: int, digest: Digest, user_ids: Iterable[int]
+) -> Mapping[int, Digest]:
     """
-    get_personalized_digests(project_id: Int, digest: Digest, user_ids: Set[Int]) -> Iterator[user_id: Int, digest: Digest]
+    TODO(mgaeta): I know this is inefficient. In the case that ProjectOwnership
+     does exist, I do the same query twice. Once with this statement and again
+     with the call to ProjectOwnership.get_actors(). I "will" follow up with
+     another PR to reduce the number of queries.
     """
-    from sentry.mail.adapter import ActionTargetType
-
-    # TODO(LB): I Know this is inefficient.
-    # In the case that ProjectOwnership does exist, I do the same query twice.
-    # Once with this statement and again with the call to ProjectOwnership.get_actors()
-    # Will follow up with another PR to reduce the number of queries.
     if (
         target_type == ActionTargetType.ISSUE_OWNERS
         and ProjectOwnership.objects.filter(project_id=project_id).exists()
     ):
         events = get_event_from_groups_in_digest(digest)
         events_by_actor = build_events_by_actor(project_id, events, user_ids)
         events_by_user = convert_actors_to_users(events_by_actor, user_ids)
-        for user_id, user_events in events_by_user.items():
-            yield user_id, build_custom_digest(digest, user_events)
+        return {
```

Comment:
does typing not like `yield`?

Reply:
I just didn't know how to type for yield but I should be able to now.